Packs/Maltiverse/Integrations/Maltiverse/Maltiverse.py

```diff
@@ -0,0 +1,504 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+from typing import Tuple, Dict, Any
+from _collections import defaultdict
+import requests
+import hashlib
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+SERVER_URL = 'https://api.maltiverse.com'
+DBOT_SCORE_KEY = 'DBotScore(val.Indicator == obj.Indicator && val.Vendor == obj.Vendor)'
+DEFAULT_THRESHOLD = 5
+
+
+class Error(Exception):
+    """Base class for exceptions in this module."""
+    pass
+
+
+class NotFoundError(Error):
+    """Exception raised for 404 - Page Not Found errors.
+
+    Attributes:
+        message -- explanation of the error
+    """
+    def __init__(self, message):
+        self.message = message
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def __init__(self, url: str, use_ssl: bool, use_proxy: bool, auth_token=None):
+        self.auth_token = auth_token
+        super().__init__(url, verify=use_ssl, proxy=use_proxy, headers={'Accept': 'application/json'})
+        if self.auth_token:
+            self._headers.update({'Authorization': 'Bearer ' + self.auth_token})
+
+    def http_request(self, method, url_suffix):
+        ok_codes = (200, 401, 403, 404, 500)  # includes responses that are ok (200) and error responses that should be
+        # handled by the client and not in the BaseClient
+        try:
+            res = self._http_request(method, url_suffix, resp_type='response', ok_codes=ok_codes)
+            if res.status_code == 200:
+                try:
+                    return res.json()
+                except ValueError as exception:
+                    raise DemistoException('Failed to parse json object from response: {}'
+                                           .format(res.content), exception)
+
+            if res.status_code in [401, 403, 500]:
+                try:
+                    err_msg = str(res.json())
+                    if self.auth_token:
+                        err_msg += ' - Check server URL and API key'
+                    else:
+                        err_msg += " - Check server URL or try using an API key"
```

Comment:
here you are adding your line in the end of the error.
please add it in the beginning of the error.
```suggestion
                        f'Check server URL and API key \n{err_msg}'
                    else:
                        f'Check server URL or try using an API key \n{err_msg}'
```